Documentation/ceph-pool-crd.md

```diff
@@ -54,7 +54,7 @@ High performance applications typically will not use erasure coding due to the p
 
 When creating an erasure-coded pool, it is highly recommended to create the pool when you have **bluestore OSDs** in your cluster
 (see the [OSD configuration settings](ceph-cluster-crd.md#osd-configuration-settings). Filestore OSDs have
-[limitations](http://docs.ceph.com/docs/luminous/rados/operations/erasure-code/#erasure-coding-with-overwrites) that are unsafe and lower performance.
+[limitations](http://docs.ceph.com/docs/nautilus/rados/operations/erasure-code/#erasure-coding-with-overwrites) that are unsafe and lower performance.
```

Comment:
Shouldn't we always point to master instead of changing the release every now and then?

Reply:
I think I would lean toward using master as well. My reasoning being that the master docs will inform the user that the docs are for a dev version, and they will then be prompted to seek out docs for the Ceph version they're using. Otherwise we have assumed they are using nautilus when they may be using mimic, and they won't see any kind of warning there.

Reply:
I still think using master is easier for the reasons mentioned by @BlaineEXE. At some point, they should look at the doc corresponding with their Ceph version.

Reply:
Did this just get missed? It's still on `nautilus`.